linux/generate_tar.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# locate config file
+CONFIG_FILE="gcp-config.yaml"
+if [ ! -e "$CONFIG_FILE" ]
+then 
+    echo "missing required config file: $CONFIG_FILE"
+    exit 1
+fi
+
+# get the repo into local
+git clone https://github.com/JingboWangGoogle/opentelemetry-collector-contrib
```

Comment:
Why not just build the code in this repo?

Reply:
Changed to, instead of cloning the repo, directly build the binary here.